mantidproject/mantid
# Restructured Absorbtion correction GUI

This PR changes the layout of the Monte Carlo Absorption correction interface slightly. It also removes an uneeded box from the cylinder shape case. See #21870.

**To test:**
1. Navigate to Interfaces > Indirect > Corrections > Calculate Monte Carlo Absorptions.
2. Check that in the Beam details box height is to the left of Width
3. Check that for each of the shapes available height is on the top line and the rest of the layout is sensible.

Then test the interface still works in the cylinder case.

1. Supply the sample field with a reduced (\*\_red) file; test files provided below.
2. Supply the container field with a reduced (\*\_red) file; test files provided below.
3. Fill in the remaining fields with the following set of properties:
    - **Cylinder**
        - _BeamHeight_: 3.5
        - _BeamWidth_: 4.0
        - _EventsPerPoint_: 200
        - _SampleChemicalFormula_: H2-0
        - _SampleDensityType_: Mass Density
        - _SampleDensity_: 1.0
        - _ContainerChemicalFormula_: Al
        - _ContainerDensityType_: Mass Density
        - _ContainerDensity_: 1.0
        - _Height_: 2.0
        - _SampleRadius_: 0.5
        - _ContainerInnerRadius_: 1.0
        - _ContainerOuterRadius_: 2.0
   
4. Run the algorithm and ensure an \*\_acc and \*\_ass workspace are created and grouped in a WorkspaceGroup.

**Test files:**
[test-files.zip](https://github.com/mantidproject/mantid/files/1305819/test-files.zip)


Fixes #21870 . 

**Release Notes** 
Have updated release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
